src/utils/vr-caps-detect.js

```diff
@@ -39,12 +42,20 @@ const GENERIC_ENTRY_TYPE_DEVICE_BLACKLIST = [/cardboard/i];
 // - gearvr: Oculus GearVR
 //
 export async function getAvailableVREntryTypes() {
-  const isWebVRCapableBrowser = !!navigator.getVRDisplays;
   const isSamsungBrowser = browser.name === "chrome" && navigator.userAgent.match(/SamsungBrowser/);
   const isOculusBrowser = navigator.userAgent.match(/Oculus/);
+  const isChromium = navigator.userAgent.match(/Chromium/);
+  const isSupermedium = navigator.userAgent.match(/supermedium/i);
+
+  // This needs to be kept up-to-date with the latest browsers that can support VR and Hubs.
+  // Checking for navigator.getVRDisplays always passes b/c of polyfill.
```

Comment:
perhaps there is a better way to do this? i didn't see any `isPolyfilled` mechanisms on `navigator.getVRDisplays`